tests/h/presenters/group_json_test.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+
+from h.presenters.group_json import GroupJSONPresenter, GroupsJSONPresenter
+
+
+class TestGroupJSONPresenter(object):
+    def test_private_group_asdict_no_urls(self, factories):
+        group = factories.Group(name='My Group',
+                                pubid='mygroup')
+
+        presenter = GroupJSONPresenter(group)
+
+        assert presenter.asdict() == {
+            'name': 'My Group',
+            'id': 'mygroup',
+            'type': 'private',
+            'public': False,
+            'scoped': False,
+            'urls': {}
+        }
+
+    def test_open_group_asdict_no_urls(self, factories):
+        group = factories.OpenGroup(name='My Group',
+                                    pubid='mygroup')
+
+        presenter = GroupJSONPresenter(group)
+
+        assert presenter.asdict() == {
+            'name': 'My Group',
+            'id': 'mygroup',
+            'type': 'open',
+            'public': True,
+            'scoped': False,
+            'urls': {}
+        }
+
+    def test_private_group_asdict_with_urls(self, factories):
+        group = factories.Group(name='My Group',
+                                pubid='mygroup')
+        route_url = mock.Mock(return_value='/group/a')
```

Comment:
These mocks won't crash if the code calls them with the wrong number of args or with keyword args that don't exist. Not a big deal here though